src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,66 @@
+package duke;
+
+import duke.list.List;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static final List LIST = new List();
+    private static final Scanner IN = new Scanner(System.in);
+    public static void main(String[] args) throws DukeException {
+        String input= "";
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+
+        System.out.println("Hello from\n" + logo);
+        addLines("Hello! I'm Duke\n" + "What can I do for you?");
+        runCommand(input);
+    }
+
+    private static void runCommand(String input) throws DukeException {
+        while(!input.equals("bye")) {
+            input = IN.nextLine();
+            try {
+                String[] splitInput = input.split(" ", 2);
+                if(splitInput.length==1 && !input.equals("bye") && !input.equals("list")){
```

Comment:
Maybe you can try extracting out the conditions into a method to improve readability, or give a comment to let others know what this condition does.